common/src/srw_lock.c

```diff
@@ -0,0 +1,344 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#include <inttypes.h>
+#include <stdbool.h>
+
+#include "windows.h"
+
+#include "azure_c_logging/xlogging.h"
+#include "azure_c_util/gballoc.h"
```

Comment:
srwlock is an abstraction, so we should move it into the interfaces/win32 folders for now.

Reply:
I forgot to remove srw_lock from common when I added it to win32. I have made these corrections there.

---
In reply to: [437081811](https://github.com/Azure/azure-c-pal/pull/1#discussion_r437081811) [](ancestors = 437081811)